symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/stream/util/StreamUtil.java

```diff
@@ -0,0 +1,42 @@
+package com.symphony.bdk.core.service.stream.util;
+
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+import org.apache.commons.codec.binary.Base64;
```

Comment:
Would it be possible to use Base64 from the JDK? (just wondering as a way to remove the commons code dependency)

Reply:
yes I can use the java.util.Base64